### ISSUE TITLE
Error creating skeleton file

### ISSUE THREAD

**Issue by 174n** [MitchWeaver/Discline#21]
If I try to copy config skeleton I always get an error.

I use Arch btw

**Comment by mitchweaver** [comment]
Known errors where this is occurring, 

1. does the `~/.config` folder exist?

2. for some people, they need to `--store-token first`, looking into why this is

if neither of these fixes it for you, join the dev discord (link in the readme), and we'll try to debug further.

Settings related stuff will be ironed out upon the rewrite

**Comment by 174n** [comment]
It starts now, but doesn't show any servers

**Comment by mitchweaver** [comment]
Have you tried `/servers`?

**Comment by h1z1** [comment]
Possibly related to this, had the same thing.  Don't know python enough to dig more past despite installing python36 and going through a bit of dependency hell, it still doesn't work.   Not blaming you though :)   After digging a bit  I don't trust the official Discord client at _all_ (and neither should anyone) which had me looking for alternatives.

Two of the errors are related to config.  Was too tired, turns out I created a directory which broke it.    "--copy-skeleton" should be done or at least the error trapped on first start.  Some rather oddly misleading errors about  not finding the term module.

I was finally able to get connected but like above the server and channel list are empty (default is I'm assuming yours?).    Typing /servers brings up a list but /channels crashes.  *shrug*

                                                                                                             
                                                                                                                     
                                                                                                                     
                                                                                                                     
```
Task exception was never retrieved
future: <Task finished coro=<input_handler() done, defined at /ssd/git/Discline/input/input_handler.py:64> exception=AttributeError("'NoneType' object has no attribute 'channels'",)>
Traceback (most recent call last):
  File "/ssd/git/Discline/input/input_handler.py", line 202, in input_handler
    await print_userlist()
  File "/ssd/git/Discline/utils/print_utils/userlist.py", line 68, in print_userlist
    if len(gc.client.get_current_server().channels) == 0:
AttributeError: 'NoneType' object has no attribute 'channels'


```

**Comment by mitchweaver** [comment]
```
AttributeError: 'NoneType' object has no attribute 'channels'
```

Ah, one of your servers is returning as null for some reason. Do you have a custom 'default_server' in your config? Try keeping it to the default, (the dev server). 

Might you know of any weird servers you're in? Perhaps with unicode in the name, or maybe with no channels, etc?

**Comment by h1z1** [comment]
Not that I can see from the web or current discord client.  Weird thing is I looks like I"m in the dev channel but it's empty (no users show)

Edit: unless you did a ninja update I didn't see, it appears to work now?  That or discord itself was having problems...  ??

**Comment by mitchweaver** [comment]
does anything happen when you try to change? `/s some_server` or perhaps channel, `/c bug_reports`

Also try `/servers` and `/channels`, check if everything looks normal

**Comment by h1z1** [comment]
Appears OK.   I literally did nothing.   Maybe still a bug if their backend screws up ?   Or some weird python thing.  Seriously it was throwing exceptions all over.

**Comment by mitchweaver** [comment]
Ah, the bug that fixes itself. Welcome to software development haha.

It could also have just been a server outage, which has been happening lately.

I'll leave this open in case it comes up again. Cheers.

**Comment by DevonJSmith** [comment]
I have the a very similar issue to h1z1. I installed correctly and was able to create the skeleton file, but when running the program I'm joined to the discline server, but it shows no users online and no channels. 

`/servers` will list servers, but trying to change to any of them with /server <name> does nothing

`/channels` throws a nasty looking exception that ends with 
`AttributeError: 'NoneType' object has no attribute 'channels'`
This to me suggests that although the UI suggests I am joined to the discline server and channel "test_bed", I am not actually correctly joined, hence why I cannot switch servers or retrieve channels.

I have both Python 3.6.5 and Python 2.7.15rc1, though I used pip3 to install and python3 to run discline.py.

Could this happen if my Discord token is incorrect?
